File: server/lib/fakers_api/people/person.ex
defmodule FakersApi.People.Person do
  use Ecto.Schema
  import Ecto.Changeset
  alias FakersApi.People.{PersonAddress, PersonContact, DeceasedPerson}

  schema "person" do
    field :birth_date, :date
    field :first_name, :string
    field :last_name, :string
    field :pesel, :string
    field :second_name, :string
    field :sex, :string

    has_many :person_addresses, PersonAddress
    has_many :addresses, through: [:person_addresses, :address]

    has_many :person_contacts, PersonContact
    has_many :contacts, through: [:person_contacts, :contact]

    has_one :deceased_person, DeceasedPerson
  end

  @doc false
  def changeset(person, attrs) do
    person
    |> cast(attrs, [:pesel, :first_name, :second_name, :last_name, :sex, :birth_date])
    |> validate_required([:pesel, :first_name, :last_name, :sex, :birth_date])
    |> unique_constraint(:pesel)
  end
end

File: generator/EntityGenerator/model/Person.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace EntityGenerator.model
{
    public partial class Person
    {
        public Person()
        {
            PersonAddresses = new HashSet<PersonAddress>();
            PersonContacts = new HashSet<PersonContact>();
        }

        public long Id { get; set; }
        public string Pesel { get; set; }
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public string LastName { get; set; }
        public char Sex { get; set; }
        public DateTime BirthDate { get; set; }

        public virtual DeceasedPerson DeceasedPerson { get; set; }
        public virtual ICollection<PersonAddress> PersonAddresses { get; set; }
        public virtual ICollection<PersonContact> PersonContacts { get; set; }
        
        public override string ToString() => $"{Id} {Pesel} {FirstName} {SecondName} {LastName} {Sex} {BirthDate}";
    }
}
